queries/scala/highlights.scm

```diff
@@ -172,7 +172,7 @@
   "sealed"
   "private"
   "protected"
-] @type.qualifier
+] @modifier
```

Comment:
was this renamed upstream? Change needs to be reflected also in CONTRIBUTING.md

Reply:
Ahh, didn't realise these were being synced from upstream. Ideally we should think about modifying nvim-treesitter to pull the upstream queries instead of maintaining a copy. That'll avoid over polluting rtp for non-installed parsers.

Reply:
Not possible in general (certainly not before we fix the query order). But the goal is to minimize the necessary changes for Neovim.

Reply:
(Not putting queries for uninstalled parsers in rtp is something we should do regardless, even if the queries still live in some other directory of this repo.)